deeprank2/utils/parsing/__init__.py

```diff
@@ -65,7 +65,11 @@ def get_vanderwaals_parameters(self, atom: Atom) -> VanderwaalsParam:
                     type_ = action["TYPE"]
 
         if type_ is None:
-            _log.warning(f"Atom {atom} is unknown to OPLS, the forcefield used in deeprank2; vanderwaals_parameters set to (0.0, 0.0, 0.0, 0.0)")
+            _log.warning(
+                f"Atom {atom} is unknown to the forcefield, vanderwaals_parameters set to (0.0, 0.0, 0.0, 0.0).\n \
+                    This will affect `vanderwaals` feature.\n \
+                        Check https://deeprank2.readthedocs.io/en/latest/features.html#nonbond-energies for more details.",
```

Comment:
As it was, each tab will get printed to the warning message

```suggestion
                f"Atom {atom} is unknown to the forcefield, vanderwaals_parameters set to (0.0, 0.0, 0.0, 0.0).\n"
                "   This will affect `vanderwaals` feature.\n"
                "       Check https://deeprank2.readthedocs.io/en/latest/features.html#nonbond-energies for more details.",
```